Increasing precision for avg pool

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#23906 Increasing precision for avg pool**

The 1-off error is expected for the average pool due to double rounding.
Increasing unittest precision tolerance to 1.0 to avoid failing.

Differential Revision: [D16678044](https://our.internmc.facebook.com/intern/diff/D16678044)

Commit:
Increasing precision for avg pool

The 1-off error is expected for the average pool due to double rounding.
Increasing unittest precision tolerance to 1.0 to avoid failing.

jerryzh168 reviewed (changes requested):
actually this should be for `int_repr` only right? could you check these separately? I think we still want to make sure q_scale(), q_zero_point() and q_scheme() match with default precision

zafartahirov commented:
> actually this should be for `int_repr` only right? could you check these separately? I think we still want to make sure q_scale(), q_zero_point() and q_scheme() match with default precision

I agree --will check

Commit:
Update on "Increasing precision for avg pool"


The 1-off error is expected for the average pool due to double rounding.
Increasing unittest precision tolerance to 1.0 to avoid failing.

Differential Revision: [D16678044](https://our.internmc.facebook.com/intern/diff/D16678044)

Commit:
Update on "Increasing precision for avg pool"


The 1-off error is expected for the average pool due to double rounding.
Increasing unittest precision tolerance to 1.0 to avoid failing.

Differential Revision: [D16678044](https://our.internmc.facebook.com/intern/diff/D16678044)

Commit:
Update on "Increasing precision for avg pool"


The 1-off error is expected for the average pool due to double rounding.
Increasing unittest precision tolerance to 1.0 to avoid failing.

Differential Revision: [D16678044](https://our.internmc.facebook.com/intern/diff/D16678044)
